modules/java/generator/gen_java.py

```diff
@@ -1018,15 +1018,15 @@ def finalize(self, output_jni_path):
         self.save(list_file, '\n'.join(['#include "%s"' % f for f in self.cpp_files]))
 
 
-def copy_java_files(java_files_dir, java_base_path):
+def copy_java_files(java_files_dir, java_base_path, default_package_path='org/opencv/'):
     global total_files, updated_files
     java_files = []
     re_filter = re.compile(r'^.+\.(java|aidl)(.in)?$')
     for root, dirnames, filenames in os.walk(java_files_dir):
        java_files += [os.path.join(root, filename) for filename in filenames if re_filter.match(filename)]
     java_files = [f.replace('\\', '/') for f in java_files]
 
-    re_package = re.compile(r'^package +(.+);$')
+    re_package = re.compile(r'^package +(.+);')
```

Comment:
Probably we should use `open(src, 'rU')` to match all line ending variants and convert them to '\n'.